.github/workflows/check-coverage-deploy.yaml

```diff
@@ -56,27 +123,40 @@ jobs:
 
       - run: ls
       - run: ls ${PKG_NAME}
-      - run: ls drat
       - run: echo ${{ github.ref }}
-      - name: Check to deploy
-        if: ${{ endswith(github.ref, '/master') || endswith(github.ref, '/main')  }}
+
+      - name: Set environmental variables 
         run: |
           cd ${PKG_NAME}
           Rscript \
             -e 'dsc = read.dcf("DESCRIPTION")' \
-            -e 'cat(sprintf("PKG_VERSION=%s\n", dsc[,"Version"]))'
-            -e 'cat(sprintf("PKG_TARBALL=%s_%s.tar.gz\n", dsc[,"Package"], dsc[,"Version"]))' >> ${GITHUB_ENV}
-          echo $PKG_TARBALL
-          echo ${PKG_TARBALL}
-          cd ../drat
+            -e 'system(sprintf("echo \"PKG_VERSION=%s\" >> $GITHUB_ENV", dsc[,"Version"]))' \
+            -e 'system(sprintf("echo \"PKG_TARBALL=%s_%s.tar.gz\" >> $GITHUB_ENV", dsc[,"Package"], dsc[,"Version"]))'
+
+      - name: Checkout drat repo
+        uses: actions/checkout@v2
+        with:
+          repository: hugheylab/drat
+          path: ${{ github.event.repository.name }}/drat
+          token: ${{ secrets.JAKEJH_PAT }}
+          
+
+      - name: Check to deploy
+        run: |
+          cd ${PKG_NAME}
+          echo "Version1: $PKG_VERSION"
+          echo "Version2: ${PKG_VERSION}"
+          echo "Tarball1: $PKG_TARBALL"
+          echo "Tarball1: ${PKG_TARBALL}"
```

Comment:
Tarball2?

Reply:
I'm confused by lack and presence of braces.

Reply:
Revising per your request in later comment.